benchmarks/synchrobench/src/main/java/com/oath/oak/synchrobench/maps/OakMap.java

```diff
@@ -93,7 +93,7 @@ public boolean descendOak(K from, int length) {
     private boolean createAndScanView(com.oath.oak.OakMap<MyBuffer, MyBuffer> subMap, int length) {
         Iterator iter;
         if (Parameters.zeroCopy) {
-            iter = subMap.zc().keySet().iterator();
+            iter = subMap.zc().keyStreamSet().iterator();
```

Comment:
Can you please add some new parameter tot he entire script? So we might run both regular iterator and the stream one?

Reply:
Will do!